src/golem-network.ts

```diff
@@ -1,4 +1,4 @@
-import { DataTransferProtocol, DeploymentOptions, GolemDeploymentBuilder } from "./deployment";
+import { DataTransferProtocol, DeploymentOptions, GolemDeploymentBuilder } from "./experimental/deployment";
```

Comment:
If deployment is moved to experimental then there should be a comment on `GolemNetwork.creteDeploymentBuilder` that it's experimental, or that method should be removed altogether

Reply:
Pull changes from beta before doing this because after moving this file to `src/golem-network/` git thinks it's a new file